ftw/envoy-config.yaml

```diff
@@ -39,7 +39,7 @@ static_resources:
                               "rules": [
                                 "Include @recommended-conf",
                                 "Include @ftw-conf",
-                                "Include crs-setup.conf.example",
+                                "Include @crs-setup-conf-example",
```

Comment:
Any thoughts on removing the word example from the keyword if it's more of a recommendation? Or we can merge into `@recommended-conf`?

Reply:
+1

Reply:
My thoughts are:
- `recommended-conf` itself is misleading. It has a list of recommended configs that the user has to tune up, they are not recommended as is.
- `crs-setup.conf.example` better express this concept with the `example` keyword: engine status, paranoia level, detection mode are all up to the user. This is the reason why in a usual installation it is required to rename `crs-setup.conf.example` into `crs-setup.conf` and properly tweak it.
- coraza conf and crs conf  are quite different configs, an user may wish to enable basic coraza configurations but not enables the crs. Also in terms of inlining rules, keeping them split permits adding custom rules between the two files. Order matters, it could be useful.

If we are providing customized configurations that we believe are suitable for coraza-proxy-wasm, the keywords that I'm most comfortable with are: `coraza.conf` and `crs-setup.conf` (or even better adding a prefix like `wasm-`. Otherwise, if it is just about providing the unmodified files (like we are doing), it could be better to stick with their original names (therefore keeping `recommended` and `example` in their names).

Reply:
I think I prefer consistency here and `@recommended-conf` is what people is mainly using (besides the change in the `SecRuleEngine` directive which can be done by adding it afterwards. Other than that I would not differ too much the coraza-proxy-wasm files from coraza as they reflect what users are used to I would just tailor them as for the proxy-wasm case (e.g. recommended-conf should comment out the directives around filesystem as they are going to be ignored anyways)

Reply:
The `-example` suffix has been removed. We are going to have `@recommended-conf` and `@crs-setup-conf` for respectively coraza and crs basic config files. Thanks!